apps/meteor/server/modules/notifications/notifications.module.ts

```diff
@@ -110,6 +111,16 @@ export class NotificationsModule {
 				return false;
 			}
 
+			if (room.t === 'c' && this.userId) {
+				if (await Authorization.hasPermission(this.userId, 'preview-c-room')) {
+					const subscription = await Subscriptions.findOneByRoomIdAndUserId(room._id, this.userId);
+
+					return !!subscription;
+				}
+
+				return false;
```

Comment:
```suggestion
				if (!(await Authorization.hasPermission(this.userId, 'preview-c-room'))) {
					const subscription = await Subscriptions.findOneByRoomIdAndUserId(room._id, this.userId);

					return !!subscription;
				}
```

Permission OR subscription.